src/Components/Movies.js

```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+import { Component } from "react";
+
+class Movies extends Component {
+  state = { titles: [], title: "", releaseDate: "", description: "" };
+
+  componentDidMount() {
+    this.fetchTitle();
+  }
+
+  fetchTitle = async () => {
+    try {
+      const res = await axios.get(`https://ghibliapi.herokuapp.com/films`);
+      const ghibliMovies = res.data;
+      this.setState({ titles: ghibliMovies });
+    } catch (error) {
+      console.log(error);
+      this.setState({ titles: [] });
+    }
+  };
+
+  handleSelect = async (e) => {
+    try {
+      const res = await axios.get(
+        `https://ghibliapi.herokuapp.com/films/${e.target.value}`
+      );
+      const ghibliTile = res.data.title;
```

Comment:
I think it's a better approach to keep track on the entire movie in the state and then you can just access the title and other pieces when you need them